docs/review.md

```diff
@@ -0,0 +1,67 @@
+# Review Guidelines
+
+The following summarises review guidelines that we follow for pull requests in
+Riot Web and other supporting repos. These are just guidelines (not strict
+rules) and may be updated over time.
+
+## Code Review
+
+When reviewing code, here are some things we look for and also things we avoid:
+
+### We review for
+
+* Correctness
+* Performance
+* Accessibility
+* Security
+* Comments and documentation where needed
+* Sharing knowledge of different areas among the team
+* Ensuring it's something we're comfortable maintaining for the long term
+
+### We should avoid
+
+* Style nits that are already handled by the linter
```

Comment:
This would be a good time for us to review our style guide/lint rules to ensure they match what we expect. Some things aren't covered, like our preference on how to indent things that span multiple lines.

Reply:
Yes, agreed! I don't have a comprehensive list of what's not covered, as I tend to only notice the wrongness as it happens, so if anyone does, let's update the rules or else keep track of things as we notice them for a rule update.